Jenkinsfile_CNP

```diff
@@ -89,9 +89,13 @@ def checkForNoCleanupLabel(branch_name) {
   return new GithubAPI(this).getLabelsbyPattern(branch_name, "cleanup").contains("no_cleanup")
 }
 
+// Configure branches to sync with master branch
+def branchesToSync = ['demo', 'ithc', 'perftest']
```

Comment:
demo need to remove

Reply:
I have added skipped envs in HighLevelDataSetupApp class, where its added demo as a skipped env